integration/integration_test.sh

```diff
@@ -5,3 +5,4 @@ INTEGRATION_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 runTest map_integration_test.sh "Map integration test"
 runTest log_integration_test.sh "Log integration test"
 runTest ct_integration_test.sh "CT integration test"
+runTest ct_multi_test.sh "CT multi-server integration test"
```

Comment:
I could go either way -- you're right that it's a superset, but also I can imagine a situation where the 1+1 test works but the 3+3 test fails, and that would be helpful information.

How about having a single script, but with parameters to indicate how many of each to run, and then have the overall script run:
```
runTest "ct_integration_test.sh 1 1" "CT integration test"
runTest "ct_integration_test.sh 3 3" "CT multi-server integration test"
```
?

(If we go for this, I'll do a separate pull request for it)